### PR TITLE
Added project documentation

### PR DESCRIPTION
**Pull request by hadassahj**
### Pull Request Overview

Added detailed project description, motivation, and architecture overview.

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs further updates.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Hadasa Jercău <hadasa.jercau@stud.fils.upb.ro>

### CODE REVIEW

**Review by TeodoraMiu — CHANGES_REQUESTED** [review]
Please add a block diagram with the main hardware components used (not a Kicad schematic, but just the main components and how they interconnect). Also, at the "Hardware" section, add a list of the components that you will need (sensors, microcontroller etc.)

File: docs/project/2024/ma.upb.ro_hadassahj/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| [TSL2561 Light Intensity Sensor Module](https://www.optimusdigital.ro/ro/senzori-senzori-optici/137-senzor-de-intensitate-a-luminii.html?search_query=0104110000000590&results=1) | Measures light intensity | [22,99 lei](https://www.optimusdigital.ro/ro/senzori-senzori-optici/137-senzor-de-intensitate-a-luminii.html?search_query=0104110000000590&results=1) |
+| [Soil_moisture_module] (https://www.optimusdigital.ro/ro/senzori-senzori-de-umiditate/73-senzor-de-umiditate-a-solului.html?search_query=0104110000001023&results=1) | Measures soil moisture levels | [3,99 lei](https://www.optimusdigital.ro/ro/senzori-senzori-de-umiditate/73-senzor-de-umiditate-a-solului.html?search_query=0104110000001023&results=1) |
+| [Jumpers] (https://www.optimusdigital.ro/ro/fire-fire-nemufate/899-set-de-fire-pentru-breadboard-rigide.html?search_query=0104110000008046&results=1) | Measures soil moisture levels | [12,49 lei](https://www.optimusdigital.ro/ro/fire-fire-nemufate/899-set-de-fire-pentru-breadboard-rigide.html?search_query=0104110000008046&results=1) |
+
+
+
+## Software
```
Comment (alexandruradovici) [inline docs/project/2024/ma.upb.ro_hadassahj/index.md:94]: No libraries?

File: docs/project/2024/ma.upb.ro_hadassahj/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Project Name
+
+PicoW Garden Monitor
+
+**Author**: JERCĂu Hadasa-Ștefana \
```
[the remark this answers is not in the document]
Comment (hadassahj) [reply docs/project/2024/ma.upb.ro_hadassahj/index.md]: Typo solved, thank you!

File: docs/project/2024/ma.upb.ro_hadassahj/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+## Software
+
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [core::panic::PanicInfo](https://doc.rust-lang.org/core/panic/struct.PanicInfo.html) | Panic handling utilities | I'll utilize this module for managing panics, accessing information about the panic such as file and line numbers. |
+| [core::str::from_utf8](https://doc.rust-lang.org/core/str/fn.from_utf8.html) | UTF-8 string conversion | This module provides functions for converting byte slices to UTF-8 strings, which will be handy for working with text data stored as byte slices. |
```
Comment (alexandruradovici) [inline docs/project/2024/ma.upb.ro_hadassahj/index.md:100]:
```suggestion
```

These are from the Rust core library.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — COMMENTED** [review]
Please rename the folder to your username.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — CHANGES_REQUESTED** [review]
Please delete old folder (ma.upb.ro_dasassahj)

**Comment by hadassahj** [comment]
> Please delete old folder (ma.upb.ro_dasassahj)

how can i do this? i only renamed the folder

**Comment by hadassahj** [comment]
> > > Please delete old folder (ma.upb.ro_dasassahj)
> > 
> > 
> > how can i do this? i only renamed the folder
> 
> I assume that you made your changes locally and pushed them on github. If so, to delete the old folder :
> 
> 1. Make sure you are on your branch.
> 2. Open the repo in a file explorer/code editor and check if the old folder is still present. (delete it if  it is still there)
> 3. If the old folder is not present locally, commit and push your changes to [your remote branch on github](https://github.com/hadassahj/upb-fils-ma.github.io/tree/project_documentation/docs/project/2024).
> 4. Make sure the folder is not present on [remote](https://github.com/hadassahj/upb-fils-ma.github.io/tree/project_documentation/docs/project/2024) and that you have no errors when pushing your changes.
> 
> If you have errors when pushing changes to the remote branch, let me know.

I think i managed to delete it, thank you for the help!

**Review by irina-b-dev — DISMISSED** [review]

**Review by TeodoraMiu — APPROVED** [review]

File: docs/project/2024/hadasa.jercau/index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+# Project Name
+
+PicoW Garden Monitor
```
Comment (alexandruradovici) [inline docs/project/2024/hadasa.jercau/index.md:3]:
```suggestion
# PicoW Garden Monitor
```

Please add a one line description of the project under the title.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/hadasa.jercau/index.md (hunk 4 above)
Comment (hadassahj) [reply docs/project/2024/hadasa.jercau/index.md]: hope i managed to solve it, git gave me a hard time

File: docs/project/2024/hadasa.jercau/index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Motivation
+
+The motivation behind PicoW Garden Monitor stems from a personal need to address common challenges faced in plant care. As someone who often forgets to water plants and struggles to find the optimal spot with the right amount of sunlight, there is a strong desire to find a solution that simplifies and enhances the gardening experience.
+
+By developing the PicoW Garden Monitor, the goal is to create a reliable and intuitive tool that helps overcome these challenges. With real-time monitoring of soil moisture, light exposure, and temperature, the system provides timely reminders for watering and insights into the ideal environmental conditions for plant growth. This empowers users to nurture healthy and thriving gardens, even with busy schedules or limited gardening experience.
+
+## Architecture 
+
+Main Components:
```
Comment (alexandruradovici) [inline docs/project/2024/hadasa.jercau/index.md:21]: Please format this, run `npm start` to see how it looks like.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/hadasa.jercau/index.md (hunk 5 above)
Comment (hadassahj) [reply docs/project/2024/hadasa.jercau/index.md:22]: tried to solve this but i think i didnt manage to
weeee

**Review by irina-b-dev — DISMISSED** [review]

File: docs/project/2024/hadasa.jercau/index.md (hunk 5 above)
Comment (alexandruradovici) [reply docs/project/2024/hadasa.jercau/index.md:22]: https://commonmark.org/help/

**Comment by hadassahj** [comment]
now it looks good, thank you for your help! hope everything is allright

**Review by TeodoraMiu — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**